src/mathematics/mod.rs

```diff
@@ -0,0 +1,58 @@
+//! The [mathematical typesetting table][1].
+//!
+//! [1]: https://www.microsoft.com/typography/otspec/math.htm
+
+mod constants;
+mod glyphinfo;
+mod variants;
+
+pub use self::constants::Constants;
+pub use self::glyphinfo::GlyphInfo;
+pub use self::variants::Variants;
+
+use layout::Device;
+
+table! {
+    @position
+    #[doc = "The mathematical typesetting table."] 
+    pub Mathematics { // Math Header Table
+        header (Header), // Math Header Table 
+
+        constants (Constants) |this, tape, position| {
+            jump_take!(tape, position, this.header.constants_offset)
+        },
+
+        glyph_info (GlyphInfo) |this, tape, position| {
+            jump_take!(tape, position, this.header.glyph_info_offset)
+        },
+
+        variants (Variants) |this, tape, position| {
+            jump_take!(tape, position, this.header.variants_offset)
+        },
+    }
+}
+
+table! {
+    #[doc = "The mathematics header table."]
+    #[derive(Copy)]
+    pub Header { // Math Header Table
```

Comment:
If a table doesn’t have a type-looking name in the spec, I don’t add any comment.